video-presentation.py

```diff
@@ -263,15 +311,17 @@ def find_slides_transitions(self, connector, host, secret_key, resource, masks=N
                 continue
 
             # Create section for file
-            sectionid = sections_upload(connector, host, secret_key, {'file_id': resource['id']})
-            slidemeta = {
-                'section_id': sectionid,
-            }
-            description = "Slide %2d at %s" % (idx + 1, datetime.timedelta(milliseconds=time_idx))
+            #sectionid = sections_upload(connector, host, secret_key, {'file_id': resource['id']})
+            #slidemeta = {
+            #    'section_id': sectionid,
+            #}
+            #description = "Slide %2d at %s" % (idx + 1, datetime.timedelta(milliseconds=time_idx))
             # upload preview & associated it with the section
-            previewid = pyclowder.files.upload_preview(connector, host, secret_key, resource['id'], slidepath, slidemeta)
+            if idx == 0:
+                pyclowder.files.upload_thumbnail(connector, host, secret_key, resource['id'], slidepath)
+            previewid = pyclowder.files.upload_preview(connector, host, secret_key, resource['id'], slidepath, {})
```

Comment:
you're skipping the whole deal with sections? I suppose you handle by using your own previewer?

Reply:
Yes, I couldn't figure out how to add on-click navigation to galleria so I did it a different way. I understand that the sections have potential value but I don't really see how to leverage it right now and removing them allowed me to tidy up the previewer a lot.

Reply:
If you don't use sections anymore, you can drop a line in the docker file: https://github.com/E-CAM/training_portal/blob/bc78cab5098030c52ec907be1df983657f9f53fd/Dockerfile#L41

Reply:
Done